### PR TITLE
Task-7294: Clean old wiki code

### PR DESCRIPTION
**Pull request by mkrout**

### CODE REVIEW

File: notes-service/src/main/java/org/exoplatform/wiki/jpa/EntityConverter.java [hunk 1]
```diff
@@ -1,3 +1,22 @@
+/*
+ * This file is part of the Meeds project (https://meeds.io/).
+ *
+ * Copyright (C) 2020 - 2021 Meeds Association contact@meeds.io
```
Comment (boubaker) [inline notes-service/src/main/java/org/exoplatform/wiki/jpa/EntityConverter.java]:
```suggestion
 * Copyright (C) 2020 - 2022 Meeds Association contact@meeds.io
```
same for other files please

File: notes-service/pom.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
   <name>eXo PLF:: Notes Service</name>
 
   <properties>
-    <exo.test.coverage.ratio>0.44</exo.test.coverage.ratio>
+    <exo.test.coverage.ratio>0.42</exo.test.coverage.ratio>
   </properties>
```
Comment (boubaker) [inline notes-service/pom.xml:15]: please cleanup the dependencies of this artifact to keep only used ones, such as https://github.com/Meeds-io/notes/blob/7d4362d9ce5cc57336b77bed1b8c3813f5dfd263/notes-service/pom.xml#L84-L105

File: notes-service/src/main/java/org/exoplatform/wiki/model/DraftPage.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+@Data
+public class DraftPage extends Page {
+  private String targetPageId;
+
+  private String targetPageRevision;
+
+  private boolean newPage;
+
+  private DiffResult changes;
```
Comment (boubaker) [inline notes-service/src/main/java/org/exoplatform/wiki/model/DraftPage.java:34]: overall code format

File: notes-service/src/main/java/org/exoplatform/wiki/model/ImportList.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+
+
+  public ImportList(long exportDate, List<Page> notes) {
+    this.exportDate = exportDate;
+    this.notes = notes;
+  }
+
+
+
```
Comment (boubaker) [inline notes-service/src/main/java/org/exoplatform/wiki/model/ImportList.java]: multiple spaces

File: notes-service/src/main/java/org/exoplatform/wiki/model/PageHistory.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+package org.exoplatform.wiki.model;
+
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+public class PageHistory {
+
+	private Long versionNumber;
```
Comment (boubaker) [inline notes-service/src/main/java/org/exoplatform/wiki/model/PageHistory.java]: please avoid using tabs (overall code format)

File: notes-service/pom.xml (hunk 2 above)
Comment (mkrout) [reply notes-service/pom.xml:15]: Done

File: notes-packaging/src/main/assemblies/notes-addon-package.xml [hunk 6]
```diff
@@ -42,24 +42,8 @@
       <includes>
         <include>${project.groupId}:*:jar</include>
         <include>commons-configuration:commons-configuration:jar</include>
```
Comment (boubaker) [inline notes-packaging/src/main/assemblies/notes-addon-package.xml:44]: image
apparently this dependency isn't included, same as for (net.sf.json-lib) https://github.com/Meeds-io/notes/pull/540/files#diff-b97cf76091695f7a02119d50c025ed92c53613d4368e8d86502825052dae5899R46

File: notes-service/src/main/java/org/exoplatform/wiki/notification/listener/EditWikiListener.java [hunk 7]
```diff
@@ -55,7 +55,7 @@ private NotificationContext buildContext(WikiService wikiService, Page page) {
     Set<String> receivers = new HashSet<String>();
 
     // Task creator
-    try {
+    /*try {
```
Comment (boubaker) [inline notes-service/src/main/java/org/exoplatform/wiki/notification/listener/EditWikiListener.java]: please delete commented code (DEVSTD)

File: notes-service/src/main/java/org/exoplatform/wiki/service/NoteService.java [hunk 8]
```diff
@@ -359,9 +361,42 @@ DraftPage createDraftForExistPage(DraftPage draftNoteToSave,
 
   NoteToExport getParentNoteOf(NoteToExport note) throws WikiException;
 
-  void importNotes(String zipLocation, Page parent, String conflict, Identity userIdentity) throws WikiException, IllegalAccessException, IOException;
+    String getNoteRenderedContent(Page note);
+
+    void importNotes(String zipLocation, Page parent, String conflict, Identity userIdentity) throws WikiException, IllegalAccessException, IOException;
```
Comment (boubaker) [inline notes-service/src/main/java/org/exoplatform/wiki/service/NoteService.java]: code format

File: notes-service/src/test/java/org/exoplatform/wiki/service/TestWikiService.java [hunk 9]
```diff
@@ -37,7 +42,11 @@
 import java.util.*;
 
 @SuppressWarnings("deprecation")
-public class TestWikiService extends BaseTest {
+public class
+
+
+
+TestWikiService extends BaseTest {
```
Comment (boubaker) [inline notes-service/src/test/java/org/exoplatform/wiki/service/TestWikiService.java]: code format

File: pom.xml [hunk 10]
```diff
@@ -16,7 +16,6 @@
   <modules>
     <module>notes-service</module>
     <module>notes-webapp</module>
-    <module>notes-webui</module>
```
Comment (boubaker) [inline pom.xml:19]: please delete the dependency management entry as well
```
      <dependency>
        <groupId>${project.groupId}</groupId>
        <artifactId>notes-webui</artifactId>
        <version>${project.version}</version>
      </dependency>
```

**Review by boubaker — APPROVED** [review]

**Pull request merged**